app/views/state_file/questions/eligible/edit.html.erb

```diff
@@ -19,12 +19,7 @@
   <% if current_state_code == "md" %>
     <%= render partial: "state_file/questions/eligible/md_reveals" %>
   <% else %>
-    <div class="reveal">
-      <p><button class="reveal__button"><%= t(".credits_not_supported")%></button></p>
-      <div class="reveal__content">
-          <%= render partial: "state_file/questions/eligible/#{current_state_code}_credits_unsupported" %>
-      </div>
-    </div>
+    <%= render partial: "state_file/questions/eligible/#{current_state_code}_credits_unsupported" %>
```

Comment:
the `reveal__button` text is something different for `nc` now, so making it go into the respective partials seemed like a good solution